debian/control

```diff
@@ -0,0 +1,62 @@
+Source: couchdb
+Section: misc
+Priority: optional
+Maintainer: CouchDB Developers <dev@couchdb.apache.org>
+Standards-Version: 3.9.6
+Build-Depends: debhelper (>= 9),
+               dh-exec,
+               dh-systemd (>= 1.5),
+               erlang-dev (>= 1:16.b.3),
```

Comment:
I went off of what is in our configure/Makefile script. We still tolerate 16B03.